### ISSUE TITLE
Request to possibly add a cost to auto-loot.

### ISSUE THREAD

**Issue by clampi** [fireundubh/fo4-mods#9]
Problem Description
While the mod is a godsend, I personally like to have a cost associated with anything that is useful, as a role-player. Currently the cost is only player-wise (download and install the mod), there is no in-game character cost. The character simply benefits.

While I enjoy using it, I also wish, if possible, for a cost to be added to that enjoyment, such that it will feel realistic.

So here is some ideas borrowed from a similar mod for FNV.
Imagine that the gathering is done by a robot. That robot would need energy, That robot is made in a pos-war era, when technical aspects of things are kept secret by the knowledgeable folks (BoS & Institute). Thus, to make a robot work, one has to assemble parts without much precision, parts that generate radiation when working together.

To sum it up
- Add energy cost, refill with fusion cells/power cores/settlement with generators
- Add a radiation cost, with the changes Fo4 has made, that is costly as one looses maximum HP and has to cleanse the blood.

**Comment by fireundubh** [comment]
The cost is carry weight. If you cheat that, then yeah, there's no cost.

Auto Loot  was inspired by [Auto Harvest 2 for Skyrim](http://www.nexusmods.com/skyrim/mods/19618), and was always intended to serve a utility role, not a mechanical one. While I think adding more complexity would be interesting, I think more complexity would overburden the purpose of the mod, and personally, I'd just turn that stuff off. With Auto Loot as-is, I'm more inclined to explore the world without hunting for tin cans and manually harvesting every single Brain Fungus in the mole rat den near Red Rocket. For me, Auto Loot makes the game more fun without introducing any new, potentially tedious mechanics.
